kyuubi-server/src/main/scala/org/apache/kyuubi/events/KyuubiOperationEvent.scala

```diff
@@ -60,7 +61,8 @@ case class KyuubiOperationEvent private (
     sessionUser: String,
     sessionType: String,
     kyuubiInstance: String,
-    metrics: Map[String, String]) extends KyuubiEvent {
+    metrics: Map[String, String],
+    progress: OperationProgress) extends KyuubiEvent {
```

Comment:
Why do we need to record progress in event? I think progress should be fetched in real time instead of recorded in event.

Reply:
https://github.com/apache/kyuubi/blob/master/kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala#L43-L63

Reply:
Now `KyuubiOperationEvent` is also used for this REST api.

Reply:
Let me try to use the dto. KyuubiOperationEvent

Reply:
`KyuubiOperationEvent` is only updated when the operation status changes, so it is not real-time.